lib/hexpm/web/views/api/retirement_view.ex

```diff
@@ -4,11 +4,21 @@ defmodule Hexpm.Web.API.RetirementView do
   def render("show." <> _, %{retirement: retirement}) do
     render_one(retirement, __MODULE__, "show")
   end
+  def render("minimal." <> _, %{retirement: retirement}) do
+    render_one(retirement, __MODULE__, "minimal")
+  end
 
   def render("show", %{retirement: retirement}) do
     %{
       message: retirement.message,
       reason: retirement.reason,
     }
   end
+
+  def render("minimal", %{retirement: %{retirement: retirement}}) when is_nil(retirement), do: %{}
```

Comment:
fyi: `%{retirement: retirement}) when is_nil(retirement)` can be written simply as `%{retirement: nil})`